docs/user/user/metrics/validation-checklist.md

```diff
@@ -0,0 +1,30 @@
+# Validating the collected data
+
+Once instrumentation is added to a product, it is worth investing time early to understand if the data looks as expected and reasonable, while clients start sending it, and act if it does not.
+
+The following checklist could help guide such validation effort.
+
+1. Before releasing the product with the new data collection, make sure the data looks as expected by generating sample data on a local machine and inspecting it on the [Glean Debug View](../../reference/debug/debugViewTag.md):
+
+    a. is the data showing up in the correct ping(s)?
+
+    b. does the metric report the expected data?
+
+    c. if exercising the same path again, it is expected for the data to be submitted again? And does it?
+
+2. As users start adopting the version of the product with the new data collection, the initial data coming in should be checked, to understand how the measurements are behaving in the wild:
+
+    a. does the shape of the collected data fit our prior expectations?
```

Comment:
Which "our"?

Is this point "Does this organcially-sent data satisfy they same quality expectations the manually-sent data did in Step 1"?